packages/editor-ui/src/plugins/telemetry/index.ts

```diff
@@ -164,12 +164,12 @@ export class Telemetry {
 
 	trackNodesPanel(event: string, properties: IDataObject = {}) {
 		if (this.rudderStack) {
-			properties.nodes_panel_session_id = this.userNodesPanelSession.sessionId;
+			properties.nodes_panel_session_id = this.userNodesPanelSession.pushRef;
```

Comment:
```suggestion
			properties.nodes_panel_push_ref = this.userNodesPanelSession.pushRef;
```
Maybe not necessary since this is telemetry